packages/hardhat-core/src/internal/hardhat-network/stack-traces/consoleLogger.ts

```diff
@@ -128,40 +118,35 @@ export class ConsoleLogger {
   /**
    * Temporary code to print console.sol messages that come from EDR
    */
-  public log(message: Buffer): void {
-    const sig = bufferToInt(message.slice(0, 4));
-    const parameters = message.slice(4);
+  public getDecodedLogs(messages: Buffer[]): string[] {
+    const logs: string[] = [];
 
-    const types = this._consoleLogs[sig];
-    if (types === undefined) {
-      return;
+    for (const message of messages) {
+      const log = this._maybeConsoleLog(message);
+      if (log !== undefined) {
+        logs.push(consoleLogToString(log));
+      }
     }
 
-    const consoleLogs = [this._decode(parameters, types)];
-
-    for (const log of consoleLogs) {
-      this._replaceNumberFormatSpecifiers(log);
-    }
+    return logs;
 
-    for (const log of consoleLogs) {
-      if (log === undefined) {
-        console.log("");
-        return;
-      }
+    // if (consoleLogs === undefined) {
+    //   console.log("");
+    //   return;
+    // }
 
-      // special case for console.log()
-      if (log.length === 0) {
-        console.log("");
-        return;
-      }
+    // // special case for console.log()
+    // if (consoleLogs.length === 0) {
+    //   console.log("");
+    //   return;
+    // }
 
-      console.log(util.format(log[0], ...log.slice(1)));
-    }
+    // console.log(util.format(consoleLogs[0], ...consoleLogs.slice(1)));
```

Comment:
Should these be removed?

Reply:
Addressed in https://github.com/NomicFoundation/hardhat/pull/4781/commits/4dc2638b1b77e757733abf63340d6ff75f882eb2